Dockerfile

```
@@ -239,6 +242,7 @@ RUN apk --update --no-cache add \
     sox \
     tar \
     tzdata \
+    udns-dev \
```

Should be `udns` for the final image

```suggestion
    udns \
```

---

Resolved. Tests passed.